### DISCUSSION TITLE
[1.18.2] Fabric port

### DISCUSSION THREAD

**Discussion by RedstoneWizard08** [lehjr/MachineMusePowersuits#80] [General]
**Is your feature request related to a problem? Please describe.**
I would like to be able to use Modular Powersuits with Fabric in my new 1.18.2 modpack, but there is no port at the moment.

**Describe the solution you'd like**
I would like there to be a Fabric verison of this mod available for 1.18.2.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by warior456** [comment]
yes please

**Comment by lehjr** [comment]
No. Not doing Fabric

**Comment by RedstoneWizard08** [comment]
Ok.

**Comment by lehjr** [comment]
2 reasons for why I will not be doing a fabric port, Fabric uses an “everything is a core mod” approach, and too much drama within the Fabric ecosystem (see QuiltMC for example).
